spec/support/capybara_helpers.rb

```diff
@@ -22,6 +22,14 @@ def choose_answer(question, choice)
     end
   end
 
+  def choose_answer_all(question, choice)
```

Comment:
I added this method to select all instances on form (where there are duplicates) and set a value (required for forms like the business financials form where there is duplication in label text). Alternatively, if we want to be specific we would need to look into adding ids to form elements so we can target specific elements